diff --git a/.config/hypr/hyprland.conf b/.config/hypr/hyprland.conf
index 3d5fdd6..0dab2cb 100644
--- a/.config/hypr/hyprland.conf
+++ b/.config/hypr/hyprland.conf
@@ -178,10 +178,10 @@ bind = $mainMod, Q, killactive,
 bind = $mainMod, F, togglefloating,
 bind = $mainMod, O, togglesplit
 
-bind = $mainMod, Y, movefocus, l
-bind = $mainMod, H, movefocus, d
-bind = $mainMod, A, movefocus, u
-bind = $mainMod, E, movefocus, r
+bind = $mainMod, H, movefocus, l
+bind = $mainMod, J, movefocus, d
+bind = $mainMod, K, movefocus, u
+bind = $mainMod, L, movefocus, r
 bind = $mainMod, Left, movefocus, l
 bind = $mainMod, Right, movefocus, r
 bind = $mainMod, Up, movefocus, u
@@ -214,23 +214,23 @@ bind = $mainMod SHIFT, S, movetoworkspace, special:scratch
 bind = $mainMod, C, togglespecialworkspace, note
 bind = $mainMod SHIFT, C, movetoworkspace, special:note
 
-bind = $mainMod ALT, E, workspace, e+1
-bind = $mainMod ALT, Y, workspace, e-1
+bind = $mainMod ALT, L, workspace, e+1
+bind = $mainMod ALT, H, workspace, e-1
 bind = $mainMod ALT, Left, workspace, -1
 bind = $mainMod ALT, Right, workspace, +1
 
-bind = $mainMod SHIFT, E, movetoworkspace, +1
-bind = $mainMod SHIFT, Y, movetoworkspace, -1
+bind = $mainMod SHIFT, L, movetoworkspace, +1
+bind = $mainMod SHIFT, H, movetoworkspace, -1
 
-bind = $mainMod CTRL, Y, resizeactive, -40 0
-bind = $mainMod CTRL, H, resizeactive, 0 40
-bind = $mainMod CTRL, A, resizeactive, 0 -40
-bind = $mainMod CTRL, E, resizeactive, 40 0
+bind = $mainMod CTRL, H, resizeactive, -40 0
+bind = $mainMod CTRL, J, resizeactive, 0 40
+bind = $mainMod CTRL, K, resizeactive, 0 -40
+bind = $mainMod CTRL, L, resizeactive, 40 0
 
-bind = $mainMod SHIFT CTRL, Y, movewindow, l
-bind = $mainMod SHIFT CTRL, H, movewindow, d
-bind = $mainMod SHIFT CTRL, A, movewindow, u
-bind = $mainMod SHIFT CTRL, E, movewindow, r
+bind = $mainMod SHIFT CTRL, H, movewindow, l
+bind = $mainMod SHIFT CTRL, J, movewindow, d
+bind = $mainMod SHIFT CTRL, K, movewindow, u
+bind = $mainMod SHIFT CTRL, L, movewindow, r
 
 bindm = ALT, mouse:272, movewindow
 
@@ -240,10 +240,10 @@ bind = $mainMod SHIFT, TAB, changegroupactive, b
 bind = $mainMod CTRL, TAB, movegroupwindow
 bind = $mainMod CTRL SHIFT, TAB, movegroupwindow, b
 
-bind = $mainMod ALT CTRL, Y, movewindoworgroup, l
-bind = $mainMod ALT CTRL, H, movewindoworgroup, d
-bind = $mainMod ALT CTRL, A, movewindoworgroup, u
-bind = $mainMod ALT CTRL, E, movewindoworgroup, r
+bind = $mainMod ALT CTRL, H, movewindoworgroup, l
+bind = $mainMod ALT CTRL, J, movewindoworgroup, d
+bind = $mainMod ALT CTRL, K, movewindoworgroup, u
+bind = $mainMod ALT CTRL, L, movewindoworgroup, r
 
 bindel = ,XF86AudioRaiseVolume, exec, wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%+
 bindel = ,XF86AudioLowerVolume, exec, wpctl set-volume @DEFAULT_AUDIO_SINK@ 5%-
diff --git a/.config/rofi/bluetooth-menu.rasi b/.config/rofi/bluetooth-menu.rasi
index 99e3ea4..d76f3c9 100644
--- a/.config/rofi/bluetooth-menu.rasi
+++ b/.config/rofi/bluetooth-menu.rasi
@@ -4,18 +4,18 @@ configuration {
     font: "JetBrainsMono Nerd Font 10";
 
     /* keys have to be unbound before they can be reused */
-    kb-accept-entry: "Return,KP_Enter";
+    kb-accept-entry: "Ctrl+m,Return,KP_Enter";
+    kb-remove-to-eol: "";
     kb-remove-char-back: "BackSpace,Shift+BackSpace";
+    kb-mode-complete: "";
     kb-secondary-copy: "";
     kb-remove-char-forward: "";
     kb-remove-to-sol: "";
-    kb-move-front: "";
-    kb-move-end: "";
     /* better vim controls */
-    kb-mode-previous: "Y,Ctrl+y";
-    kb-row-down: "H,Ctrl+h,Down";
-    kb-row-up: "A,Ctrl+a,Up";
-    kb-mode-next: "E,Ctrl+e";
+    kb-mode-previous: "H,Ctrl+h";
+    kb-row-down: "J,Ctrl+j,Down";
+    kb-row-up: "K,Ctrl+k,Up";
+    kb-mode-next: "L,Ctrl+l";
     kb-cancel: "Ctrl+c,Escape";
     kb-page-next: "Page_Down,Ctrl+d";
     kb-page-prev: "Page_Up,Ctrl+u";
diff --git a/.config/rofi/config.rasi b/.config/rofi/config.rasi
index 8541caa..379293e 100644
--- a/.config/rofi/config.rasi
+++ b/.config/rofi/config.rasi
@@ -4,18 +4,18 @@ configuration {
     sidebar-mode: true;
 
     /* keys have to be unbound before they can be reused */
-    kb-accept-entry: "Return,KP_Enter";
+    kb-accept-entry: "Ctrl+m,Return,KP_Enter";
+    kb-remove-to-eol: "";
     kb-remove-char-back: "BackSpace,Shift+BackSpace";
+    kb-mode-complete: "";
     kb-secondary-copy: "";
     kb-remove-char-forward: "";
     kb-remove-to-sol: "";
-    kb-move-front: "";
-    kb-move-end: "";
     /* better vim controls */
-    kb-mode-previous: "Y,Ctrl+y";
-    kb-row-down: "H,Ctrl+h,Down";
-    kb-row-up: "A,Ctrl+a,Up";
-    kb-mode-next: "E,Ctrl+e";
+    kb-mode-previous: "H,Ctrl+h";
+    kb-row-down: "J,Ctrl+j,Down";
+    kb-row-up: "K,Ctrl+k,Up";
+    kb-mode-next: "L,Ctrl+l";
     kb-cancel: "Ctrl+c,Escape";
     kb-page-next: "Page_Down,Ctrl+d";
     kb-page-prev: "Page_Up,Ctrl+u";
diff --git a/.config/rofi/power-menu.rasi b/.config/rofi/power-menu.rasi
index 7242af6..e075d8c 100644
--- a/.config/rofi/power-menu.rasi
+++ b/.config/rofi/power-menu.rasi
@@ -4,18 +4,18 @@ configuration {
     font: "JetBrainsMono Nerd Font 10";
 
     /* keys have to be unbound before they can be reused */
-    kb-accept-entry: "Return,KP_Enter";
+    kb-accept-entry: "Ctrl+m,Return,KP_Enter";
+    kb-remove-to-eol: "";
     kb-remove-char-back: "BackSpace,Shift+BackSpace";
+    kb-mode-complete: "";
     kb-secondary-copy: "";
     kb-remove-char-forward: "";
     kb-remove-to-sol: "";
-    kb-move-front: "";
-    kb-move-end: "";
     /* better vim controls */
-    kb-mode-previous: "Y,Ctrl+y";
-    kb-row-down: "H,Ctrl+h,Down";
-    kb-row-up: "A,Ctrl+a,Up";
-    kb-mode-next: "E,Ctrl+e";
+    kb-mode-previous: "H,Ctrl+h";
+    kb-row-down: "J,Ctrl+j,Down";
+    kb-row-up: "K,Ctrl+k,Up";
+    kb-mode-next: "L,Ctrl+l";
     kb-cancel: "Ctrl+c,Escape";
     kb-page-next: "Page_Down,Ctrl+d";
     kb-page-prev: "Page_Up,Ctrl+u";
diff --git a/.config/rofi/wifi-menu.rasi b/.config/rofi/wifi-menu.rasi
index 9587778..092dc64 100644
--- a/.config/rofi/wifi-menu.rasi
+++ b/.config/rofi/wifi-menu.rasi
@@ -4,18 +4,18 @@ configuration {
     font: "JetBrainsMono Nerd Font 10";
 
     /* keys have to be unbound before they can be reused */
-    kb-accept-entry: "Return,KP_Enter";
+    kb-accept-entry: "Ctrl+m,Return,KP_Enter";
+    kb-remove-to-eol: "";
     kb-remove-char-back: "BackSpace,Shift+BackSpace";
+    kb-mode-complete: "";
     kb-secondary-copy: "";
     kb-remove-char-forward: "";
     kb-remove-to-sol: "";
-    kb-move-front: "";
-    kb-move-end: "";
     /* better vim controls */
-    kb-mode-previous: "Y,Ctrl+y";
-    kb-row-down: "H,Ctrl+h,Down";
-    kb-row-up: "A,Ctrl+a,Up";
-    kb-mode-next: "E,Ctrl+e";
+    kb-mode-previous: "H,Ctrl+h";
+    kb-row-down: "J,Ctrl+j,Down";
+    kb-row-up: "K,Ctrl+k,Up";
+    kb-mode-next: "L,Ctrl+l";
     kb-cancel: "Ctrl+c,Escape";
     kb-page-next: "Page_Down,Ctrl+d";
     kb-page-prev: "Page_Up,Ctrl+u";
diff --git a/.tmux.conf b/.tmux.conf
index 7df4966..7a75472 100644
--- a/.tmux.conf
+++ b/.tmux.conf
@@ -20,20 +20,20 @@ set-option -g detach-on-destroy off
 unbind r
 bind r source-file ~/.tmux.conf \; display "Config reloaded!"
 
-unbind -n M-L
-unbind -n C-M-h
-unbind -n C-M-l
-unbind -n M-J
-unbind -n M-K
-unbind -n M-\"
-bind -n M-Y previous-window
-bind -n M-E next-window
-bind -n C-M-y swap-window -t -1 \; previous-window
-bind -n C-M-e swap-window -t +1 \; next-window
-bind -n M-H switch-client -n
-bind -n M-A switch-client -p
-bind -n M-I switch-client -l
-bind -n M-: last-window
+unbind -n M-Y
+unbind -n M-E
+unbind -n C-M-y
+unbind -n C-M-e
+unbind -n M-A
+unbind -n M-I
+bind -n M-H previous-window
+bind -n M-L next-window
+bind -n C-M-h swap-window -t -1 \; previous-window
+bind -n C-M-l swap-window -t +1 \; next-window
+bind -n M-J switch-client -n
+bind -n M-K switch-client -p
+bind -n M-: switch-client -l
+bind -n M-\" last-window
 
 bind-key x kill-pane
 bind-key X kill-session
